bin/disco_cloudfront.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+"""
+Manages Cloudfront
+"""
+from __future__ import print_function
+import argparse
+import sys
+from disco_aws_automation import read_config
+from disco_aws_automation import DiscoCloudfront, DiscoVPC
+from disco_aws_automation.disco_aws_util import run_gracefully, is_truthy
+from disco_aws_automation.disco_logging import configure_logging
+
+
+# R0912 Allow more than 12 branches so we can parse a lot of commands..
+# pylint: disable=R0912
+def get_parser():
+    '''Returns command line parser'''
+    parser = argparse.ArgumentParser(description='Asiaq Cloudfront Creation and Management')
+    parser.add_argument('--debug', dest='debug', action='store_const', const=True, default=False,
+                        help='Log in debug level')
+    subparsers = parser.add_subparsers(help='Sub-command help')
+
```

Comment:
Just a generic question why arguments like --debug are not in the ArgumentParser class , aren't they used for all commands ?